packages/web-app-files/src/components/SideBar/Shares/Collaborators/InviteCollaborator/InviteCollaboratorForm.vue

```diff
@@ -392,8 +467,18 @@ export default defineComponent({
 .role-selection-dropdown {
   max-width: 150px;
 }
+
 #new-collaborators-form-create-button {
   padding-left: 30px;
   padding-right: 30px;
 }
+
+#show-more-actions-btn {
+  margin-left: $oc-space-small;
+  margin-right: $oc-space-small;
+}
+
+oc-datepicker {
+  width: 100%;
+}
```

Comment:
Does this work? 🤔 Because it's not an element that exists in the DOM.

Reply:
no, has no effect, just a leftover... this would work in Angular ;-)